server/port.js

```diff
@@ -1,3 +1,4 @@
-const argv = require('./argv');
+// const argv = require('./argv');
 
-module.exports = parseInt(argv.port || process.env.PORT || '3000', 10);
+// module.exports = parseInt(argv.port || process.env.PORT || '3000', 10);
+module.exports = parseInt('8080', 10);
```

Comment:
Please don't change this code if we don't have to. Looks like we just need to have an .env file with PORT=8080 in the root of the project to make this work. Add a .env file to the repo

Reply:
I thought about adding the .env, but that would require everyone remembering to do that.  So, I guess I would add that to the readme too?  Or check that into github?